test/unit/node.js

```diff
@@ -208,6 +208,17 @@ describe('VASTLoader', function () {
       )
     })
 
+    it('throws VAST 100 on invalid VAST XML while using strict parsing', async function () {
```

Comment:
Actually, it should throw a VAST 101 in this case.

It should throw a VAST 100 if it is invalid XML. Say an empty body or a JSON file.